src/viper/pretty.ml

```diff
@@ -152,9 +152,10 @@ and pp_stmt' ppf = function
       pp_exp exp1
       pp_seqn s1
       pp_seqn s2
-  | WhileS (exp, _, s) -> (* TODO: Invariant *)
-    fprintf ppf "@[<v 2>while (%a) {@ %a}@]"
+  | WhileS (exp, invs, s) ->
+    fprintf ppf "@[<v 2>while (%a) @[<v 0>%a@] {@ %a}@]"
```

Comment:
```suggestion
    fprintf ppf "@[<v 2>while (%a)@;@[<v 0>%a@]@;%a@]"
```

This removes spare brackets and move invariants on next line

Reply:
Thank you, yes, this makes the output cleaner